src/Postmark/PostmarkClient.php

```diff
@@ -90,9 +91,9 @@ function sendEmail($from, $to, $subject, $htmlBody = NULL, $textBody = NULL,
 	 * @param  array $metadata  Add metadata to the message. The metadata is an associative array , and values will be evaluated as strings by Postmark.
 	 * @return DynamicResponseModel
 	 */
-	function sendEmailWithTemplate($from, $to, $templateId, $templateModel, $inlineCss = true,
+	function sendEmailWithTemplate($from, $to, $templateId, $templateAlias = NULL, $templateModel, $inlineCss = true,
```

Comment:
@pgraham3 This change would break existing user's code if they aren't careful about semantic versioning. Instead, `$templateId` should be changed to `$templateIdOrAlias` and the method should adjusted to detect whether the value is a string or number and assign it to the correct field of the payload.

Reply:
@atheken I'll get that fixed - thanks!